src/navbar/navbar.tsx

```diff
@@ -179,52 +210,107 @@ const Component = <T,>(
     const renderNavbar = () => {
```

Comment:
this function is getting too long. could it be broken up into smaller render functions with `renderBrand()`, `renderDesktopNavItems()`, `renderDesktopNavActions()` etc.?